### PR TITLE
[#5172] Health check time is always in the Pacific timezone 

### PR DESCRIPTION
**Pull request by jaydeepkumara**
Backend:- Removed hardcoded US timezone and changed to UTC.
Frontend:- Getting UTC time and converting to the local timezone.

### CODE REVIEW

**Review by SergeyPotachev — APPROVED** [review]
Looks good to me. But I'm not UI developer, don't know a lot of details about js.

File: managed/ui/src/components/universes/UniverseDetail/compounds/UniverseHealthCheckList.js [hunk 1]
```diff
@@ -225,7 +224,7 @@ const detailsFormatter = (cell, row) => {
 function prepareData(data) {
   return data.map((timeDataJson) => {
     const timeData = JSON.parse(timeDataJson);
-    const timestampMoment = moment(timeData.timestamp);
+    const timestampMoment = momentTimezone.utc(timeData.timestamp).local();
```
Comment (sshev) [inline managed/ui/src/components/universes/UniverseDetail/compounds/UniverseHealthCheckList.js:227]: @jaydeepkumara are you sure about `momentTimezone`?
it seems `moment.utc()` is what we need
https://momentjs.com/docs/#/parsing/utc/

Comment (jaydeepkumara) [reply managed/ui/src/components/universes/UniverseDetail/compounds/UniverseHealthCheckList.js]: changed to a moment. Both libraries support UTC fun. The earlier moment had given an error so had used the moment timezone but I changed the import statement and it worked now.

**Review by sshev — APPROVED** [review]

**Comment by sshev** [comment]
Make sure to add `Platform:` prefix into the PR title pls

**Pull request merged**